my.css

```diff
@@ -352,3 +352,650 @@ body {
+}
+.total {
+  width: 475px;
+  padding: 46px 20px 45px 55px; 
+  margin: 0 auto;
+  display: flex;
+  flex-flow: column;
+  background-color: #fae8e4;
+  border: 1px solid #fae8e4;
+  box-sizing: border-box;
+}
+.total__text {
+  padding-top: 5px;
+  padding-bottom: 30px;
+  margin: 0 auto;
+  opacity: 0.5;
+}
+.total__choice {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 40px;
+  padding-top: 14px;
+}
+
+.total__choice-buttons {
+  border: 1.5px solid #000;
+  width: 95px;
+  height: 40px;
+  display: flex;
+  justify-content: space-around;
+}
+.total__choice-buton {
+  box-sizing: border-box;
+  background-color: #fae8e4;
+  border: #fff;
+}
+.total__choice-figure {
+  display: flex;
+  align-items: center;
+  
+}
+.total__basket{
+ display: flex;
+ width: 283px;
+ height: 44px;
+ align-items: center;
+ justify-content: center;
+}
+.total__basket-buttons {
+  width: 100%;
+  height: 43px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  background-color: #fae8e4;
+}
+.sectionlock__footer {
+  display: flex;
+  justify-content: flex-end;
+  padding-top: 30px;
+}
+.sectionlock__footer-img {
+  padding-left: 15px;
+}
+.footer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+@media screen and (max-width: 1130px) {
+     .sectionlock {
+    max-height: 900px;
+    max-width: 1150px;
+    position: relative;
+     }
+     .sectionlock__img-clothes {
+  width: 700px;
+  height: 640px;
+    }
+     .sectionlock__iner {
+     width: 35%;
+    }
+     .total {
+  width: 375px;
+  padding: 36px 15px 35px 45px;
+    }
+    .total__heading_color {
+      font-size: 25px;
+    }
+    .total__subheading {
+      font-size: 14px;
+    }
+    .total__text_font {
+      font-size: 13px;
+    }
+    .sectionlock__text {
+      font-size: 10px;
+    }
+  .total__choice {
+       padding-top: 10px;
+    }
+   .total__basket-buttons { 
+      height: 35px;
+      border: 1px solid #000;
+    }
+        .total__choice-buttons {
+     border: 1px solid #000;
+   width: 90px;
+   height: 33px;
+    }
+    .total__basket{
+    width: 203px;
+    }
+}
+
+@media screen and (max-width: 815px) {
+   .sectionlock__img-clothes {
+  width: 600px;
+  height: 540px;
+    } 
+}
+ @media screen and (max-width: 768px) {
+   .sectionlock {
+    width: 100%;
+    max-height: unset;
+    margin: 0 auto;
+    background: linear-gradient(90deg,
+     rgba(235,227,216,1) 11%, rgba(235,227,216,1) 16%,
+      rgba(235,227,216,1) 22%, rgba(0,0,0,1) 22%,
+       rgba(0,0,0,1) 34%, rgba(235,227,216,1) 34%);
+  }
+  .sectionlock__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .sectionlock__img-clothes {
+     width: 400px;
+     height: 340px;
+     margin: 0;
+     padding: 0;
+      }
+      .sectionlock__img {
+      width: 70%;
+      height: 444px;
+      display: flex;
+      
+   }
+   .sectionlock__iner { 
+     position: relative;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)